schema/artist/index.js

```diff
@@ -462,13 +472,15 @@ const ArtistType = new GraphQLObjectType({
               is_reference: true,
               solo_show: false,
               at_a_fair: false,
+              visible_to_public: false,
               size: options.size,
             }),
             // Fair booths
             gravity('related/shows', {
               artist_id: id,
               sort: '-end_at',
               at_a_fair: true,
+              visible_to_public: false,
```

Comment:
Am I reading this correctly that MP makes **9** different requests to the same API endpoint (granted, with different combinations of parameters) to fill out the exhibition highlights?